src/main/export/export.ts

```diff
@@ -1,53 +1,71 @@
 /* eslint-disable no-plusplus */
 import { BrowserWindow } from 'electron';
-import { writeFileSync } from 'fs';
+import { existsSync, writeFileSync } from 'fs';
```

Comment:
If the whole function is async anyway better to use `fs/promises` so that the calls are non blocking

Reply:
Done!